build.go

```diff
@@ -44,9 +44,8 @@ var (
 	// buildVersion allows override of the version string from env variable
 	buildVersion = env("BUILD_VERSION", "")
 
-	grpcProtocVersion = env("GRPC_PROTOC_VER", "3.7.1")      // version of protoc
-	grpcGogoProtoTag  = env("GRPC_GOGO_PROTO_TAG", "v1.2.1") // gogo version tag
-	grpcGatewayTag    = env("GRPC_GATEWAY_TAG", "v1.8.5")    // grpc gateway version tag
+	grpcProtocVersion = env("GRPC_PROTOC_VER", "3.14.0")     // version of protoc
+	grpcGogoProtoTag  = env("GRPC_GOGO_PROTO_TAG", "v1.3.1") // gogo version tag
```

Comment:
I believe latest version of gogo is `v1.3.2` - why not use it?

Reply:
client-go `v1.19` is using gogo `v1.3.1`. I figured it'd be safer to just keep the versions the same. https://github.com/kubernetes/client-go/blob/e9b0f31f935be48b6589f6026a2cb1891d92fe12/go.mod#L13